docs/resources/equinix_fabric_network.md

```diff
@@ -10,6 +10,23 @@ description: |-
 
 Fabric V4 API compatible resource allows creation and management of Equinix Fabric Network
 
+## Example Usage
+
+```hcl
+resource "equinix_fabric_network" "new_network"{
+  name = "Network-SV"
+  type = "EVPLAN"
+  scope = "GLOBAL"
+  notifications{
+    type = "ALL"
+    emails = ["example@equinix.com","test1@equinix.com"]
+  }
+  project {
+  	project_id = "776847000642406"
```

Comment:
nit: indentation is different for notifications and for project. Terraform fmt resolves to 2 spaces per tab. Our examples should follow the same format.

You can modify your editor settings to have it default to that for future things.

Reply:
Indentation is still off.